### PR TITLE
Feat meteor upgrade

### PR DESCRIPTION
**Pull request by andreleon**
This PR contains everything needed to upgrade to meteor 1.4 (#826)

### CODE REVIEW

File: config/development/settings.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "public": {
         "development": true,
         "aws": {
-            "bucket": "partup-development",
+            "bucket": "partup-production",
```
Comment (wouterd) [inline config/development/settings.json]: This doesn't look right...

**Comment by wouterd** [comment]
```
Apr 06 14:31:20 meteor-1 meteor-1.stg-feat-meteor-upgrade.0e4da7fa:  Note: you are using a pure-JavaScript implementation of bcrypt. 
Apr 06 14:31:20 meteor-1 meteor-1.stg-feat-meteor-upgrade.0e4da7fa:  While this implementation will work correctly, it is known to be 
Apr 06 14:31:20 meteor-1 meteor-1.stg-feat-meteor-upgrade.0e4da7fa:  approximately three times slower than the native implementation. 
Apr 06 14:31:20 meteor-1 meteor-1.stg-feat-meteor-upgrade.0e4da7fa:  In order to use the native implementation instead, run 
Apr 06 14:31:20 meteor-1 meteor-1.stg-feat-meteor-upgrade.0e4da7fa:    meteor npm install --save bcrypt 
Apr 06 14:31:20 meteor-1 meteor-1.stg-feat-meteor-upgrade.0e4da7fa:  in the root directory of your application. 
```

File: config/development/settings.json (hunk 1 above)
Comment (andreleon) [inline config/development/settings.json]: I changed it :)

**Comment by wouterd** [comment]
I have the image running on acceptance now connected to the new mongo deployment.
There's no version 1.3 of meteor that works with the new deployment model of compose, so we need to be creative here if we want to do a rolling upgrade. Have some ideas... :)

**Comment by ralphboeije** [comment]
Issues found:
- Something goes wrong with email input box on homepage (too large, causing button to be on second line) - screenshot 1
- Input on this email field is not copied to /register form email field - screenshot 2
- password fields on /register width is off - screenshot 2
- start a tribe button on homepage has strange on hoover state (colored text and underlined)

Will continue testing later today

**Comment by ralphboeije** [comment]
@andreleon I tested quite thoroughly (>90% of all features, forms and pages) and next to the comment above, these are my findings. Just css stuff I guess, features works perfect.

Other issues found:
- Private chat - padding around chat details in the left column (screenshot 1)
- Part-up settings - tags form field has strange "alt text?" position (screenshot 2)
- Part-up settings - radio button has strange grey square active state marking (screenshot 2)
- Part-up settings - date picker blue circle around date not centered around the number (screenshot 2 - NB: on activity edit date picker it is correct)
- Part-up activities - The + sign is missing on the lane insert (dotted line lane - screenshot 3)
- Tribe settings - bulk invite upload button has extra and not styled "choose file" button (screenshot 4)
- Tribe, Discover and Part-up menu, and tribe title on part-up - tabs are underlined on hover (which shouldn't be as the menu item is also underlined - screenshot 5 shows case in tribe)
- Several buttons have the same issue with underlined text on hover, but not all (see also issue in previous comment with hover state of "start a tribe" button on homepage)

screen shot 2017-04-10 at 10 10 21
 
screen shot 2017-04-10 at 10 18 18

screen shot 2017-04-10 at 10 33 16

screen shot 2017-04-10 at 10 36 22

screen shot 2017-04-10 at 10 53 30

**Comment by andreleon** [comment]
I fixed all of the listed styling issues. The upgrade to meteor 1.4 caused some packages to include new depencencies. One of which was twbs:bootstrap which included bootstrap.css.

@ralphboeije maybe one final review before merging/deploying?

**Comment by ralphboeije** [comment]
@andreleon Tnx, will do!

**Pull request merged**